plugins/scaffolder-backend/src/scaffolder/storage/index.test.ts

```diff
@@ -28,25 +29,27 @@ describe('Storage Interface Test', () => {
     };
   })();
 
+  const logger = winston.createLogger();
```

Comment:
oh that's simpler, and it just throws away the logs?

Reply:
yep!